File: projects/m1/004-units-of-time/solutions/valentinaBazzacchi/solution-units-of-time.js
const SEC_IN_MIN = 60;
const SEC_IN_HRS = 60 * SEC_IN_MIN;
const SEC_IN_DAY = 24 * SEC_IN_HRS; 
    
let daysByUser;
let hrsByUser;
let minByUser;
let secByUser;
    
daysByUser = +prompt("How many days?");
hrsByUser = +prompt("How many hours?");
minByUser = +prompt("How many minutes?");
secByUser = +prompt("How many seconds?");
    
alert( `The total number of seconds is ${(daysByUser * SEC_IN_DAY + hrsByUser * SEC_IN_HRS + minByUser * SEC_IN_MIN + secByUser)}.` );

File: projects/m1/004-units-of-time/solutions/jurgenametaj/solution_units-of_time.py
# Displays the conversion from dd,hh,mm,ss to only ss

def input_time(name):
    while True:
        try:
            x = int(input('How many '+name+'?: '))
            if x>=0:
                return x
            else:
                print('I need a positive number, try again')
        except:
            print('I need a natural number, try again')


print('Hi, I can convert days, hours, minutes and seconds in seconds')
TotSeconds = ((input_time('days')*24+input_time('hours'))*60+input_time('minutes'))*60+input_time('seconds')
print('The total number of seconds that you gave me is: '+str(TotSeconds))